apiary.apib

```diff
@@ -3683,6 +3683,11 @@ Get a list of tracked time.
                         + milestone
                         + ticket
                 + id: `2659dc4d-444b-4ced-b51c-b87591f604d7` (string, required)
+            + relates_to (object, optional) - Find all tracked time linked directly and indirectly to a subject
+                + type (enum, required)
+                    + Members
+                        + milestone
```

Comment:
Is it a lot of work to support this for other types too?
And would we benefit from making this an array immediately?

Reply:
> And would we benefit from making this an array immediately?

Would that make it an inclusive or exclusive filter (AND / OR) ?

Reply:
@stivni We are going to implement it for `project` very soon, since we are going to need that. So I made it an array (also for more consistency)

Reply:
And it should not be an array. You provide one relation in the filter. At least that's for now. Combining relations is YAGNI